api-server/api-server-common/src/storage/impls/postgres/queries.rs

```diff
@@ -2397,34 +2397,35 @@ impl<'a, 'b> QueryFromConnection<'a, 'b> {
         &self,
         len: u32,
         offset: u64,
-        ticker: &[u8],
+        ticker: &str,
     ) -> Result<Vec<TokenId>, ApiServerStorageError> {
         let len = len as i64;
         let offset = offset as i64;
+        let ticker_patern = format!("%{ticker}%");
```

Comment:
You need to escape the existing wildcard characters in `ticker` (need tests for this as well)